spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -721,6 +727,7 @@ def setup_editor(self,
                      strip_mode=False,
                      intelligent_backspace=True,
                      automatic_completions=True,
+                     completions_hint=True,

```

Comment:
This is a comment for a future PR, but we should probably explain all theses parameters within the docstring so that it is clear what each one of them is doing.(Maybe open an issues @dalthviz ?)